src/hooks/useSynth.tsx

```diff
@@ -1,14 +1,14 @@
 import React from "react";
-import {useAtom} from "jotai";
-import {synthStartedAtom, frequencyAtom} from "../store/synth";
-import {selectedTrackAtom} from "../store/tracks";
-import {ToneOscillatorType} from "tone";
-import {osc1Range} from "../store/oscillators";
+import { useAtom } from "jotai";
```

Comment:
We need to figure out why our auto-formatting settings are different 🤔

Reply:
that's weird, i thought we calibrated that a while ago